addons/html_editor/static/src/main/font/color_selector.js

```diff
@@ -91,7 +91,7 @@ export class ColorSelector extends Component {
 
     onColorPreview(ev) {
         const color = ev.hex ? ev.hex : this.processColorFromEvent(ev);
-        this.props.applyColorPreview({ color: color || "", mode: this.mode });
+        this.props.applyColorPreview({ color: color || "", mode: this.mode, previewMode: true });
```

Comment:
No need to pass it through `onColorPreview` pass it from `previewableApplyColor` rather.